packages/vsg/package.yaml

```
@@ -0,0 +1,14 @@
+name: vsg
```

```suggestion
name: vhdl-style-guide
```

The package name should match the project name first if it's unique rather than the binary name